kahuna/public/js/components/gr-image-usage-photosales/gr-image-usage-photosales.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from "react";
+import * as angular from "angular";
+import { react2angular } from "react2angular";
+import {useEffect, useState} from "react";
+import * as moment from 'moment';
+import "./gr-image-usage-photosales.css";
+
+export interface Usage {
+  title: string,
+  usageName: string,
+  usageType: string,
+  dateAdded: Date
+}
+
+export interface UsageProps {
+  usages: Usage[];
+}
+
+export interface UsagePropsWrapper {
```

Comment:
You don't need to wrap the props. You can use `UsageProps` directly, then the corresponding Angular code becomes:

`<image-usage-photo-sales usages="ctrl.processedReactUsages"></image-usage-photo-sales>`